migration.rst

```diff
@@ -0,0 +1,38 @@
+.. index::
+   single: Migration
+
+Migrating an existing application to Symfony
+============================================
+
+<!-- TODO -->
+
+When you have an existing application that was not built with Symfony,
+you might want to move over parts of that application without rewriting
+the existing logic completely. For those cases there is a pattern called
+`Strangler Application`_. The basic idea of this pattern is to create a
+new application that falls back to the existing application that needs
+to be migrated and over time the functionality can transition from the
+existing application to the new strangler application. This migration
+approach can be implemented with Symfony in various ways.
+
+General problems
+----------------
+
+<!-- TODO -->
+
+Symfony as primary system, existing application as fallback
+-----------------------------------------------------------
+
+<!-- TODO -->
+
+Existing application as primary system with Symfony as fallback
```

Comment:
I would not offer this approach. At least in my experience, this approach is always inferior to the first one if your goal is eventually having a Symfony 3/4 application.

Reply:
Yes, I think it would make more sense to discuss different approaches with the first space instead